timeseries/src/autogluon/timeseries/models/gluonts/torch/models.py

```diff
@@ -43,6 +42,7 @@
 
 class AbstractGluonTSPyTorchModel(AbstractGluonTSModel):
     gluonts_estimator_class: Type[GluonTSPyTorchLightningEstimator]
+    float_dtype: Type = np.float32
```

Comment:
I will set the dtype for DeepAR and SFF torch models for float32 for now to make sure that all tests involving dynamic features work fine. Will revisit this after running the benchmark.